### PR TITLE
[Task #1978878] Support create cosmos db account in intellij

### PR DESCRIPTION
**Pull request by Flanker32**
What does this implement/fix? Explain your changes.
---------------------------------------------------
Support create cosmos db account in intellij

Does this close any currently open issues?
------------------------------------------
[AB#1978878](https://dev.azure.com/mseng/a4d27ce2-a42d-4b71-8eef-78cee9a9728e/_workitems/edit/1978878)

Any relevant logs, screenshots, error output, etc.?
-------------------------------------

Any other comments?
-------------------
…

Has this been tested?
---------------------------
- [ ] Tested

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-cosmos/src/main/java/com/microsoft/azure/toolkit/intellij/cosmos/creation/CreateCosmosDBAccountAction.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                .create(defaultResourceGroupName, defaultResourceGroupName) : resourceGroup;
+        final CosmosDBAccountDraft.Config config = new CosmosDBAccountDraft.Config();
+        config.setName(name);
+        config.setSubscription(subscription);
+        config.setResourceGroup(group);
+        config.setKind(DatabaseAccountKind.SQL);
+        return config;
+    }
+
+    @AzureOperation(name = "kubernetes.create_service.service", params = {"config.getName()"}, type = AzureOperation.Type.ACTION)
```
Comment (wangmingliang-ms) [inline PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-cosmos/src/main/java/com/microsoft/azure/toolkit/intellij/cosmos/creation/CreateCosmosDBAccountAction.java]: `kubernetes`?

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-cosmos/src/main/java/com/microsoft/azure/toolkit/intellij/cosmos/creation/CreateCosmosDBAccountAction.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            }
+            dialog.setOkActionListener(config -> {
+                doCreate(config, project);
+                dialog.close();
+            });
+            dialog.show();
+        });
+    }
+
+    public static CosmosDBAccountDraft.Config getDefaultConfig(final ResourceGroup resourceGroup) {
```
Comment (wangmingliang-ms) [inline PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-cosmos/src/main/java/com/microsoft/azure/toolkit/intellij/cosmos/creation/CreateCosmosDBAccountAction.java:46]: is it possible to move this method to `CosmosDBAccountDraft`?

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-cosmos/src/main/java/com/microsoft/azure/toolkit/intellij/cosmos/IntelliJCosmosActionsContributor.java [hunk 3]
```diff
@@ -9,29 +9,45 @@
...
 public class IntelliJCosmosActionsContributor implements IActionsContributor {
     @Override
     public void registerHandlers(AzureActionManager am) {
+        final BiPredicate<Object, AnActionEvent> serviceCondition = (r, e) -> r instanceof AzureCosmosService;
+        final BiConsumer<Object, AnActionEvent> handler = (c, e) -> CreateCosmosDBAccountAction.create(e.getProject(), getDefaultConfig(null));
+        am.registerHandler(ResourceCommonActionsContributor.CREATE, serviceCondition, handler);
+
+        final BiConsumer<ResourceGroup, AnActionEvent> groupCreateHandler = (r, e) ->
+                CreateCosmosDBAccountAction.create(e.getProject(), getDefaultConfig(r));
+        am.registerHandler(CosmosActionsContributor.GROUP_CREATE_KUBERNETES_SERVICE, (r, e) -> true, groupCreateHandler);
```
Comment (wangmingliang-ms) [inline PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-cosmos/src/main/java/com/microsoft/azure/toolkit/intellij/cosmos/IntelliJCosmosActionsContributor.java]: `KUBERNETES`? there are many "kubernetes" in this PR.

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-cosmos/src/main/java/com/microsoft/azure/toolkit/intellij/cosmos/creation/CosmosDBAccountCreationDialog.java [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+    }
+
+    @Override
+    public AzureForm<CosmosDBAccountDraft.Config> getForm() {
+        return this;
+    }
+
+    @Override
+    protected String getDialogTitle() {
+        return "Create Cosmos DB Account";
```
Comment (wangmingliang-ms) [inline PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-cosmos/src/main/java/com/microsoft/azure/toolkit/intellij/cosmos/creation/CosmosDBAccountCreationDialog.java]: should be `Azure Cosmos DB Account`?

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-cosmos/src/main/java/com/microsoft/azure/toolkit/intellij/cosmos/creation/CreateCosmosDBAccountAction.java (hunk 1 above)
Comment (Flanker32) [reply PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-cosmos/src/main/java/com/microsoft/azure/toolkit/intellij/cosmos/creation/CreateCosmosDBAccountAction.java]: related typo has been resolved in https://github.com/microsoft/azure-tools-for-java/pull/6852/commits/3ed0f70f9241d192a82513453279ffce20d16a7b

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-cosmos/src/main/java/com/microsoft/azure/toolkit/intellij/cosmos/IntelliJCosmosActionsContributor.java (hunk 3 above)
Comment (Flanker32) [reply PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-cosmos/src/main/java/com/microsoft/azure/toolkit/intellij/cosmos/IntelliJCosmosActionsContributor.java]: related typo has been resolved in https://github.com/microsoft/azure-tools-for-java/pull/6852/commits/3ed0f70f9241d192a82513453279ffce20d16a7b

**Review by wangmingliang-ms — APPROVED** [review]
LGTM

**Pull request merged**